src/socket-to-conn.js

```diff
@@ -98,3 +96,19 @@ module.exports = (socket, options = {}) => {
 
   return maConn
 }
+
+/**
+ * Get local multiaddr from socket.
+ *
+ * @param {SimplePeer} socket
+ * @returns {Multiaddr|undefined}
+ */
+function toLocalAddr (socket) {
+  if (socket.localAddress && socket.localPort) {
+    try {
+      return toMultiaddr(socket.localAddress, socket.localPort)
+    } catch {
+      // Might fail if the socket.localAddress is fqdn
```

Comment:
@wemeetagain what about this `catch` case? Should it be handled somehow?

Reply:
If we want to ensure we get a multiaddr from a fqdn then we would need to change upstream in simple-peer and find a way to get the ip version of the connection https://github.com/ipfs-shipyard/simple-peer/blob/1039b90f91fc8e91492ff1af52f9fc9bfff97c35/index.js#L773 with a fqdn domain.

I am not convinced that is worth it as it works fine in Chrome with `undefined`.